app/org/sagebionetworks/bridge/config/BridgeConfig.java

```diff
@@ -15,10 +15,6 @@
     private static final String TEMPLATE_CONFIG = CONFIG_FILE;
     private static final String LOCAL_CONFIG = System.getProperty("user.home") + "/" + ".bridge" + "/" + CONFIG_FILE;
 
-    private static final String ENTERPRISE_STORMPATH_ID = "enterprise.stormpath.id";
-    private static final String ENTERPRISE_STORMPATH_SECRET = "enterprise.stormpath.secret";
-    private static final String ENTERPRISE_STORMPATH_APPLICATION_HREF = "enterprise.stormpath.application.href";
```

Comment:
They have already been removed from bridge-server.conf. We can remove them from heroku/AWS after this change is pushed out.